docs/docker_deployment.md

```diff
@@ -91,3 +91,27 @@ different exporters, you may find them and their documentation available at
 After updating the `otelcol-config-extras.yml`, start the demo by running
 `docker compose up`. After a while, you should see the traces flowing into
 your backend as well.
+
+## Vendor Forking
+
+Setting up a fork or a demo usually only requires overriding some environment
```

Comment:
This sentence doesn't fit well here, because the chapter "Bring your own backend" explains all this in detail.